### ISSUE TITLE
"molecule login" with single host should not require --host

### ISSUE THREAD

**Issue by MarkusTeufelberger** [metacloud/molecule#864]
# Issue Type
- Feature request

# Desired Behaviour

If only a single host is defined, it would be nice if molecule wouldn't ask for a "--host" parameter when running "molecule login".

# Actual Behaviour (Bug report only)
````
$ molecule login
Usage: molecule login [OPTIONS]

Error: Missing option "--host".
$ molecule list
Instance Name      Driver Name    Provisioner Name    Scenario Name    Created    Converged
-----------------  -------------  ------------------  ---------------  ---------  -----------
xxxxxxxxx-default  Vagrant        Ansible             default          True       True
````

**Comment by retr0h** [comment]
Implemented on v2 @ 7dea645503854b69518b17166f09ec16ab1667d1.

This bug does not report which version of Molecule was used.  However, v1 will not be getting this fix.

**Comment by MarkusTeufelberger** [comment]
Ah, sorry... Yes, it was using v2, I don't use v1 and don't expect it to get any new features anyways.

Thanks! 👍
